data/pug.txt

```diff
@@ -317,6 +317,16 @@ outfit "Pug Biodefenses"
 	"unplunderable" 1
 	description "This defense system is designed to release a suite of toxins that are lethal to human beings (and probably to some other species, as well) but which have no effect at all on the Pug."
 
+outfit "Pug Peacekeeping Staff"
+	plural "Pug Peacekeeping Staves"
+	category "Hand to Hand"
+	cost 11000
+	thumbnail "outfit/pug staff"
+	"capture attack" 0.8
+	"capture defense" 1.5
+	"unplunderable" 1
+	description "These tools designed for the control of planetary populations are also good at protecting their ships. Using minor amounts of radiation and electric shocks it non-lethally removes the complete ability to oppose the pug."
```

Comment:
Capitalize Pug.
Also, I'd say this description could be more interesting when the way it's described in the story is as follows.

>With a somewhat contemptuous gesture, the Pug points the staff toward your face and a wave of wonderful rainbow light washes over you and bathes the whole world in iridescent sparkles that leap nimbly from the whiskers of things and nuzzle into your contented consciousness like flashing minnows of pure joy. You caper off to chase a trail of floating soapy baubles that burst with the sound of wind chimes and distant trumpets as you touch each one and the smell of ripe peaches fills the air until you stagger through your airlock and throw up in a corner.

Reply:
How about "Despite being designed for planetary population control, peacekeeping staves also work exceptionally well in ship defense. The staff emits a short-ranged beam that immediately incapacitates anyone it touches by causing them to enter a euphoric and hallucinogenic state. wandering off to somewhere where they won't cause any trouble. Exiting this euphoric state is never pleasant, though, often resulting in vomiting."